### PR TITLE
Update for k8s 1.6

### PR DESCRIPTION
**Pull request by lwr20**
- Updates for k8s 1.6
- Updates to match scale test operator 0.6.0

This PR contains the updates I found were required for scale testing of Calico 2.2 with k8s 1.6.2.

### CODE REVIEW

File: gce/Makefile [hunk 1]
```diff
@@ -320,11 +320,12 @@ deploy-high-perf-nodes:
 	  $(HIGH_PERF_NODE_NAMES) \
 	  --zone $(GCE_REGION) \
 	  --image-project $(GCE_IMAGE_PROJECT) \
-	  --image $(GCE_IMAGE_NAME) \
+	  --image-family $(GCE_IMAGE_FAMILY) \
 	  --machine-type $(HIGH_PERF_INSTANCE_TYPE) \
 	  --scopes $(NODE_SCOPES) \
 	  --metadata-from-file user-data=build/node-config-template.yaml \
 	  --no-address \
+	  --boot-disk-size=100GB \
```
Comment (caseydavenport) [inline gce/Makefile]: This seems pretty big - what needs this much, prometheus?

File: gce/templates/master-config-template.yaml [hunk 2]
```diff
@@ -148,6 +148,7 @@ coreos:
         --cloud-provider=gce \
         --cloud-config=/etc/gce.conf \
         --service-cluster-ip-range=10.100.0.0/24 \
+        --storage-backend=etcd2 \
```
Comment (caseydavenport) [inline gce/templates/master-config-template.yaml]: I think we should upgrade to etcdv3 instead.

File: gce/Makefile (hunk 1 above)
Comment (lwr20) [inline gce/Makefile]: Exactly.

Comment (lwr20) [inline gce/Makefile]: Behaviour when you run out of space is pretty bad: all pods on the dead node basically stop.  Which is a pain when one of them turns out to be the test-operator.

File: gce/templates/master-config-template.yaml (hunk 2 above)
Comment (lwr20) [inline gce/templates/master-config-template.yaml]: Sure - but this worked without further modification.  Without this line, the cluster simply doesn't work - so clearly some flags need setting elsewhere.  I reckon that upgrade to etcdv3 should be a separate PR.

**Comment by lwr20** [comment]
@caseydavenport So is anything else required before this can be merged?

**Comment by caseydavenport** [comment]
@lwr20 LGTM.
